packages/app/src/client/services/AdminCustomizeContainer.js

```diff
@@ -97,10 +106,13 @@ export default class AdminCustomizeContainer extends Container {
         currentCustomizeHeader: customizeParams.customizeHeader,
         currentCustomizeCss: customizeParams.customizeCss,
         currentCustomizeScript: customizeParams.customizeScript,
+        attachmentId: customizeParams.attachmentId,
```

Comment:
Change the name to `brandLogoAttachmentId`

Reply:
- `attachmentId renamed to `brandLogoAttachmentId`